apps/llm-client/bin/prod-deploy.sh

```
@@ -0,0 +1,40 @@
+
+if [ ! -f "$OUT_DIR/index.html" ]; then
+  echo "No output files found. Run: npm run build --prefix $APP_DIR"
+  exit 1
+fi
+
+src_url="$OUT_DIR"
+dst_url="gs://kyle.pericak.com/apps/llm-client"
+
+echo "Checking for changes..."
+dry_run_output=$(gsutil -m rsync -r -c -n "$src_url" "$dst_url")
+if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)